plugins/reporters/web-app-template/src/components/IssuesTable.jsx

```diff
@@ -23,14 +23,15 @@ import {
 } from 'react';
 
 import {
-    ExclamationCircleOutlined,
     FileAddOutlined,
-    FileExcelOutlined,
-    InfoCircleOutlined,
-    IssuesCloseOutlined,
-    WarningOutlined
+    FileExcelOutlined
```

Comment:
Commit message:
- Typo in "experie*n*ce".
- Missing dot at end of sentence.
- Wrapping seems to be earlier than column 75 in the first line.